ForneverMind/posts/en/2017-11-07-multilingual-support.md

```diff
@@ -1,33 +1,34 @@
-    title: Multiple languages are supported on this site
+    title: Multilingual support on fornever.me
     description: A small note about latest developments of this site.
 ---
 
 [For a long time][issue-10] I wanted to add English language support to the
-site, and to start writing posts in English. Finally, I've found the time to
-implement this.
+site, and to start publishing posts in English. Finally, I've found the time to
+implement this. Meet, [fornever.me][] is now in English, too!
 
 This post will be the first in my blog that will be published in both English
-and Russian. I'll eventually translate all the other posts, too. Comments for
-the older posts were preserved, but the URL scheme has been changed a bit. At
-the beginning, I also wanted to preserve the redirects from the older post
-locations to the newer ones, but ultimately I've decided that it will require
-too much effort.
+and Russian. I'll eventually translate all the other posts, too. The comments
+for the older posts were preserved, but the URL addressing has been changed a
+bit. Initially I also wanted to add redirects from the old post locations to the
+new ones, but ultimately I've decided that it will require too much effort and
+provide too little benefits.
 
-I'd like to point my regular readers (if they ever exist) to the changes in RSS:
-the site now has _three_ RSS feeds.
+I'd like to point my regular readers (in case they ever exist) to the changes in
+RSS: the site now has _three_ RSS feeds.
 
 * [`/rss.xml`](/rss.xml) — the main feed than contains posts in both English and
   Russian
 * [`/en/rss.xml`](/en/rss.xml) — English only posts
 * [`/ru/rss.xml`](/ru/rss.xml) — Russian only posts
 
 Please, set up your RSS readers according to your language preferences. For a
-while, English part of the site will be pretty empty, bit I hope to bring the
-parity one day by translating all of my materials to both languages.
+while, English part of the site will be pretty empty, bit I hope to eventually
```

Comment:
"**bit** I hope" should say "but I hope".

Reply:
Yas, thanks! I'll fix it.